content/docs/get-started/aws/deploy-stack.md

```diff
@@ -55,7 +55,7 @@ Resources:
 Duration: 14s
 ```
 
-Remember the output you defined in the previous step? That [stack output]({{< relref "/docs/intro/concepts/stack#outputs" >}}) can be seen in the `Outputs:` section of your update. You can access your outputs from the CLI by running the `pulumi stack output [property-name]` command. For example you can print the name of your bucket with the following command:
+Remember the output you defined in the previous step? That [stack output]({{< ref "/docs/intro/concepts/stack#outputs" >}}) can be seen in the `Outputs:` section of your update. You can access your outputs from the CLI by running the `pulumi stack output [property-name]` command. For example you can print the name of your bucket with the following command:
```

Comment:
I think this should still be `relref`. What is the reasoning behind just using `ref` here?

Reply:
Yes, we should always use relative URLs unless we can't, for some reason.

  ```suggestion
Remember the output you defined in the previous step? That [stack output]({{< relref "/docs/intro/concepts/stack#outputs" >}}) can be seen in the `Outputs:` section of your update. You can access your outputs from the CLI by running the `pulumi stack output [property-name]` command. For example you can print the name of your bucket with the following command:
```

Reply:
Yup, this was part of some other WIP things I was doing and forgot to change this back. Good catch.